test_elasticsearch_async/conftest.py

```diff
@@ -61,17 +62,19 @@ def port():
 
 @yield_fixture
 def server(event_loop, port):
-    server = DummyElasticsearch(debug=True, keep_alive=75)
+    server = DummyElasticsearch(debug=True, loop=event_loop)
     f = event_loop.create_server(server, '127.0.0.1', port)
     event_loop.run_until_complete(f)
     yield server
     event_loop.run_until_complete(server.shutdown(timeout=.5))
 
 @yield_fixture
 def client(event_loop, server, port):
+    logger = logging.getLogger('elasticsearch')
+    logger.setLevel(logging.DEBUG)
```

Comment:
required for the testcase `test_elasticsearch_async/test_connection.py::test_request_is_properly_logged`